assets/blocks/reader-registration/index.php

```diff
@@ -279,6 +279,15 @@ function process_form() {
 		);
 	}
 
+	// reCAPTCHA test.
+	if ( ! empty( $_REQUEST['captcha_token'] ) ) {
```

Comment:
If this is not sent and reCaptcha is enabled it should fail. This allows for the captcha validation to be skipped.

Reply:
Whoops should fix this. Now the check is for `can_use_captcha` instead of whether the token is empty.